### PR TITLE
First attempt at an Applicability section

### PR DESCRIPTION
**Pull request by DavidSchinazi**

### CODE REVIEW

File: draft-iab-protocol-maintenance.md [hunk 1]
```diff
@@ -113,6 +113,19 @@ and implementations being strict in following the requirements of
 those specifications.
 
 
+## Applicability
+
+The guidance in this document applies to many types of networking protocols and
+areas of deployment, however there are exist unusual circumstances where it does
+not. In particular, the guidance here relies on the ability to update software
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:120]: I think that we want to say something about the big-I Internet.  How does this sound?

> The guidance in this document is intended for protocols that are deployed to the Internet.
> There are some situations in which this guidance might not apply to a protocol due to conditions on its implementation or deployment.
>
> In particular, this guidance depends on an ability to update and deploy implementations.
> Being able to update implementations that are deployed to the Internet is an essential part of managing security risk.
> In practice, not all protocol deployments are continuously maintained.
>
> Where updates do not occur, there is no opportunity to apply the practices this document recommends.
> In particular, some practices - such as those in {{intolerance}} - only exist to support the development of protocol maintenance and evolution.
> Employing this guidance is therefore only applicable where there is the possibility of improving deployments through updates.

Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md]: I like it, I've tweaked your text and updated the PR

File: draft-iab-protocol-maintenance.md [hunk 2]
```diff
@@ -113,6 +113,25 @@ and implementations being strict in following the requirements of
 those specifications.
 
 
+## Applicability
+
+The guidance in this document is intended for protocols that are deployed to the
+Internet. There are some situations in which this guidance might not apply to a
+protocol due to conditions on its implementation or deployment.
+
+In particular, this guidance depends on an ability to ongoingly update and
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:122]:
```suggestion
In particular, this guidance depends on an ability to update and
```

File: draft-iab-protocol-maintenance.md [hunk 3]
```diff
@@ -113,6 +113,25 @@ and implementations being strict in following the requirements of
...
+to the Internet is an essential part of managing security risk. In practice, not
+all protocol deployments are continuously maintained.
+
+Where implementations are not updated, there is no opportunity to apply the
+practices that this document recommends. In particular, some practices - such as
+those described in {{intolerance}} - only exist to support the development of
+protocol maintenance and evolution. Employing this guidance is therefore only
+applicable where there is the possibility of improving deployments through
+updates of their implementations.
+
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:133]: I wanted to leave one more question here.  We acknowledge the need to add some temporary hacks when updates are not possible in the short term.  Is this a good place to place a signpost for that possibility?

**Review by martinthomson — APPROVED** [review]
I'll merge this and submit ahead of the deadline either way.

**Pull request merged**